config/locales/collection/collection-fr.yml

```diff
@@ -0,0 +1,307 @@
+---
+fr:
+  collection:
+    approve_all:
+      approved_n_pages: "%{page_count} pages approuvées."
+    collection_works:
+      n_pages:
+        one: '1 page :'
+        other: "%{count} pages :"
+      progress: Progrès
+      work_title: Titre de travail
```

Comment:
"Work" is getting translated as "travail" (task, job, work" rather than "oeuvre".